helper_test.go

```diff
@@ -262,7 +262,7 @@ func createOAuthToken(t *testing.T, client *Client, org *Organization) (*OAuthTo
 func createOrganization(t *testing.T, client *Client) (*Organization, func()) {
 	ctx := context.Background()
 	org, err := client.Organizations.Create(ctx, OrganizationCreateOptions{
-		Name:  String(randomString(t)),
+		Name:  String("go-" + randomString(t)),
```

Comment:
this is new. it namespaces the test organizations so they're easier to find & cleanup. maxchars is 40 so this barely fits.

Reply:
I don't know if we really need this, but if we do I suggest to prefix it with `tst-` as it seems more common and can be used by other tests as well.

I took the liberty to update this in the PR.